lib/rules/one-var.js

```diff
@@ -62,20 +120,46 @@ module.exports = function(context) {
         "FunctionDeclaration": startFunction,
         "FunctionExpression": startFunction,
         "ArrowFunctionExpression": startFunction,
+        "BlockStatement": startBlock,
```

Comment:
@sebmck fixed this in babel master. These tests should pass when babel updates eslint gets the updated dep.